spec/modules/paste-html-to-govspeak.spec.js

```diff
@@ -0,0 +1,29 @@
+describe('GOVUK.Modules.PasteHtmlToGovspeak', function () {
```

Comment:
I'm a bit confused by this test working as we don't seem to actually set `GOVUK.Modules.PasteHtmlToGovspeak`. I don't think this is actually running as it is not in spec/javascripts